src/libraries/System.Formats.Cbor/src/System/Formats/Cbor/Writer/CborWriter.Simple.cs

```diff
@@ -36,7 +19,7 @@ public void WriteHalf(Half value)
         public void WriteSingle(float value)
         {
             if (!CborConformanceModeHelpers.RequiresPreservingFloatPrecision(ConformanceMode) &&
-                 FloatSerializationHelpers.TryConvertSingleToHalf(value, out Half half))
+                 TryConvertSingleToHalf(value, out var half))
```

Comment:
Shouldn't this be typed explicitly?

Reply:
Hmm presumably it's a different type depending on the target?

Reply:
Yeah, I was going to leave that comment earlier, then realized it's `Half` if netcoreapp or `float` if ns2.0